packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -476,20 +492,12 @@ function createSandbox(packageManager: string) {
   );
 
   try {
-    execSync(`${packageManager} install --silent`, {
-      cwd: tmpDir,
-      stdio: 'ignore',
-    });
+    await execAndWait(`${packageManager} install --silent`, tmpDir);
   } catch (_) {
-    // Install failed so run again without --silent
     try {
-      execSync(`${packageManager} install`, {
```

Comment:
This was only to get the logs. Now that we use `execAndWait`, we do not need to try it again.